src/styles/base/_override.typography.scss

```diff
@@ -101,17 +110,17 @@
 .ds-text-body--md,
 .mct-p3,
 .ds-text-body--sm {
+  color: $black-500;
```

Comment:
Can we use `$color-gray-dark` here instead of `$black-500` as that seems to be more of a higher level token

Reply:
This is complete.